src/sentry/models/eventerror.py

```diff
@@ -1,6 +1,20 @@
 from __future__ import absolute_import
 
 import six
+from string import Formatter
+
+
+class dontexplodedict(dict):
```

Comment:
don't subclass dict. this means we copy out the thing for no good reason. Alternatively proposal:

```python
class X(object):
 def __init__(self, x):
  self.x = x
 def __getitem__(self, x):
  try:
   return self.x[x]
  except LookupError:
   return ''
```